pkg/agent/manager/manager.go

```diff
@@ -289,25 +288,39 @@ func (m *manager) runBundleObserver(ctx context.Context) error {
 }
 
 func (m *manager) storeSVID(svidChain []*x509.Certificate) {
-	err := StoreSVID(m.svidCachePath, svidChain)
+	err := storage.StoreSVID(m.svidCachePath, svidChain)
 	if err != nil {
 		m.c.Log.WithError(err).Warn("Could not store SVID")
 	}
 }
 
+func (m *manager) storeReattestable(reattestable bool) {
+	if reattestable {
+		err := storage.StoreReattestable(m.reattestablePath)
+		if err != nil {
```

Comment:
style nitpick: when there is only an error returned and the error can be handled entirely within the scope of the if statement, it is a little more concise do do this on one line:
```suggestion
		if err := storage.StoreReattestable(m.reattestablePath); err != nil {
```